arcaflow_plugin_pcp/pcp_plugin.py

```diff
@@ -91,6 +91,17 @@ def start_pcp(
             if "error" in pmlogconf_return[0]:
                 return pmlogconf_return
 
+        # Import the pmrep configuration file, if provided
```

Comment:
This comment is kind of a misdirect:  what this code is really doing is _selecting_ the `pmrep` configuration file (i.e., it is setting `pmrep_conf_path`) -- it creates that file if the contents are supplied and otherwise uses a default file.  (That is, this code comment appears to be attached to an `if` statement but it doesn't address the `else` clause.)

Reply:
As a comment, IMO the "else" case being the default is implied. I can reword this.